src/anyio/_backends/_asyncio.py

```diff
@@ -771,6 +755,7 @@ async def __aexit__(
                                     and not is_anyio_cancellation(exc)
                                 ):
                                     exc_val = exc
+                            self._on_completed_fut = None
```

Comment:
Add a blank line after a control block.
```suggestion

                            self._on_completed_fut = None
```